packages/machine/test/integration/cf-operations.spec.ts

```diff
@@ -459,14 +444,16 @@ async function uninstallBalanceRefund(
   amountA: ethers.utils.BigNumber,
   amountB: ethers.utils.BigNumber
 ) {
-  const msg = startUninstallBalanceRefundMsg(
-    multisigAddr,
-    cfAddr,
+  const response = await walletA.runUninstallProtocol(
     walletA.signingKey.address,
     walletB.signingKey.address,
-    amountA
+    multisigAddr,
+     [
+      new cf.legacy.utils.PeerBalance(walletA.signingKey.address, amountA),
```

Comment:
I think we should remove the use of this `PeerBalance` class if we can; it doesn't really do anything useful

Reply:
This is used in a few places and I don't think there's any replacement for it; we can remove it in a separate PR